### ISSUE TITLE
briefcase is still not compatible with pip v10

### ISSUE THREAD

**Issue by jedie** [pybee/briefcase#120]
#117 doesn't fix pip v10 compatibility :(

```
  File "/opt/briefcase/briefcase/app.py", line 457, in run
    self.install_support_package()
  File "/opt/briefcase/briefcase/app.py", line 402, in install_support_package
    pip.download.unpack_url(
AttributeError: module 'pip' has no attribute 'download'
```
Full output: https://travis-ci.org/jedie/pybee-docker/builds/366740949

**Comment by freakboy3742** [comment]
Gah... I made a mess of this one. I made the changes on my fork and did a release without merging into the main repository. Should be corrected now.

**Comment by freakboy3742** [comment]
... or not. I was working off a really stale branch, so there's a bunch of other bits that need to be updated. Working on it now.

**Comment by freakboy3742** [comment]
Ok; this has now been released as v0.2.3. It's a bit of an interim fix; longer term, we shouldn't be relying on the internal API of pip.

Thanks for the report (both of them 😅)

**Comment by jedie** [comment]
Now it works, see: https://travis-ci.org/jedie/pybee-docker ... But i have still the Android SDK licence problem...
